setup.py

```diff
@@ -0,0 +1,31 @@
+from setuptools import setup, find_packages
+from pathlib import Path
+
+# Use semantic versioning
+VERSION = "0.1.0"
+
+PY_VERSION_REQUIRED = ">=3.7"
+PACKAGES_REQUIRED = {
+    "core": [],
+    "test":[
+        "pytest"
+    ]
+}
+
+# Read README.md to use as long description in package metadata
+root_dir = Path(__file__).parent.absolute()
+with (root_dir / "README.md").open("rt") as readme_file:
+    long_description = readme_file.read().strip()
+
+setup(
+    name="neffint",
+    version=VERSION,
+    description="A python package for computing Fourier integrals using a Filon type method with non-equidistant grid spacing.",
+    long_description=long_description,
+    url="https://github.com/neffint/neffint/",
+    author="Eskil Vik, Nicolas Mounet",
```

Comment:
I don't know what it proper etiquette regarding the order here, let me know if I should change it, doesn't matter to me

Reply:
:+1: